client/src/dashboard/pages/contacts/Contacts.jsx

```diff
@@ -67,9 +80,11 @@ function Contacts() {
   return (
     <div className={styles.content}>
       <h1>Contacts</h1>
-
+      <div className={styles.btnContainer}>
```

Comment:
Can we use ProtectedElement component to show the button for level 2+ only? See Organizations.jsx